tests/tests/datm_ocn_ice_mx025_gefs

```diff
@@ -0,0 +1,63 @@
+#
+#  CPLD_DATM_MOM6_CICE_CMEPS_1_STEP_COLD test
+#
+
+export TEST_DESCR="DATM-MOM6-CICE-CMEPS - 1-step cold start"
+
+export CNTL_DIR="datm_ocn_ice_mx025_gefs"
+export CNTLMED_DIR="datm_ocn_ice_mx025_gefs/RESTART"
+
+export LIST_FILES="RESTART/MOM.res.nc \
+                   RESTART/MOM.res_1.nc \
+                   RESTART/MOM.res_2.nc \
+                   RESTART/MOM.res_3.nc \
+                   RESTART/iced.2011-10-02-00000.nc \
+                   RESTART/DATM_GEFS.cpl.r.2011-10-02-00000.nc \
+                   ocn_2011_10_01_21.nc"
+
+export_datm
+export FHMAX=24  # Note this needs to be at least 2x CPL_SLOW
+export WLCLK=40
+export DATM_SRC="GEFS"
+export FILENAME_BASE='gefs.'
+export IATM=1536
+export JATM=768
+
+export atm_petlist_bounds="0 0"
+export med_petlist_bounds="0 39"
+export ocn_petlist_bounds="40 159"
+export ice_petlist_bounds="160 207"
+export TASKS=208
+export TPN=40
+export NPROC_ICE='48'
+
+# atm/ocn/ice resolution
+export OCNRES='025'
+export ICERES='0.25'
+export NX_GLB=1440
+export NY_GLB=1080
+#
+# set component and coupling timesteps
+export DT_CICE=${DT_ATMOS}
+export CPL_SLOW=${DT_THERM_MOM6}
+export CPL_FAST=${DT_ATMOS}
+
+# cice restart (days)
```

Comment:
The dumpfreq and dumpfreq_n are no longer relevant. The restart frequency is controlled by restart_n and restart_option in the nems. configure. These should be removed.

Reply:
They are removed.